code/game/objects/items/devices/scanners/autopsy_scanner.dm

```diff
@@ -0,0 +1,61 @@
+/obj/item/autopsy_scanner
+	name = "autopsy scanner"
+	desc = "Used in surgery to extract information from a cadaver."
+	icon = 'icons/obj/device.dmi'
+	icon_state = "autopsy_scanner"
+	inhand_icon_state = "autopsy_scanner"
+	worn_icon_state = "autopsy_scanner"
+	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
+	flags_1 = CONDUCT_1
+	slot_flags = ITEM_SLOT_BELT
+	w_class = WEIGHT_CLASS_SMALL
+	custom_materials = list(/datum/material/iron = 200)
+	custom_price = PAYCHECK_COMMAND
+
+/obj/item/autopsy_scanner/proc/scan_cadaver(mob/living/carbon/human/user, mob/living/carbon/scanned)
```

Comment:
if this is the tool to check dead people info, shouldnt it show oxy/tox/cloning damage as sign of poisoning or choking or whatever?